src/components/CollateralSurplusAction.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react'
+import { Button, Flex, Spinner } from '@chakra-ui/react'
+
+import { LiquityStoreState } from '@liquity/lib-base'
+import { useLiquitySelector } from '../hooks/useLiquitySelector'
+
+import { useLiquity } from '../hooks/LiquityContext'
+
```

Comment:
a quick pass across all files to remove empty lines would be great for clean code ✨